src/Umbraco.Web/Models/ContentEditing/CodeFileDisplay.cs

```diff
@@ -14,6 +14,9 @@ public class CodeFileDisplay : INotificationModel
         [DataMember(Name = "virtualPath", IsRequired = true)]
         public string VirtualPath { get; set; }
 
+        [DataMember(Name = "path", IsRequired = true)]
+        public string Path { get; set; }
```

Comment:
Can we add some .Net code comments here to explain what "Path" is? And/Or come up with a better name